### PR TITLE
[build] Update banner to say MATTER.

### PR DESCRIPTION
**Pull request by turon**
#### Problem
The `bootstrap.sh` banner says "CHIP", but the project is generally branded as "Matter" now.

#### Change overview
Change the build banner from "CHIP" to "MATTER".

#### Testing
Ran it and it worked!

### CODE REVIEW

**Review by yunhanw-google — APPROVED** [review]
Thanks

File: scripts/bootstrap.sh [hunk 1]
```diff
@@ -38,11 +38,11 @@ _bootstrap_or_activate() {
...
-  ▄███▒  ░▓█  ░▓█ ░▓█▓ ▒█████▄
- ██▒ ▀█▒  ▒█   ▒█  ░█▒  ▒█░  █░
- █▓░      ▒██████  ░█▒  ▒█▄▄▄█░
- ▓█   █▒  ▒█   ▒█  ░█░  ▒█▀
- ░▓███▀  ░▓███░▓█▒ ░█░  ▒█
+███    ███  █████  ████████ ████████ ███████ ██████  
+████  ████ ██   ██    ██       ██    ██      ██   ██ 
+██ ████ ██ ███████    ██       ██    █████   ██████  
+██  ██  ██ ██   ██    ██       ██    ██      ██   ██ 
+██      ██ ██   ██    ██       ██    ███████ ██   ██
```
Comment (jmeg-sfy) [inline scripts/bootstrap.sh:45]:
```suggestion
           ░▓░                                                                                      
           ▓█▓                                                                                      
           ▓█▓                                                     ▒█     ▒█                        
      ▒██▒▒▓██▒███▒                ░▒▓▒░  ░▒▓▒░       ░░▓█▒░ ░█  █████████████░    ░▒█▒░       ░░▒░ 
       ░▓█████▓██░               ▒█▒░░▒▓██▓▒░░▒█▒   ░█▓▒░░▒████    █▓░    █▓░    ▒█▒░░░▒█▓░   ██▒░░ 
    ▒█▒░         ░██▒           ░█░     █▓     ░█░ ░▓▒      ░▓█    █▓     █▓    ▒█░░    ░██  ▒▓     
    ░▓██▓░     ░██▓█░           ░█      ▓█      █░ ░█░       ██    █▓     █▓    ▓██████████  ▒█     
     ░▓███▒   ▒███▒░            ░█      ▓█      █░  ██░    ░███    █▓     █▓    ░█▒░         ▒█     
 ░▒████████░ ░███▓▓█▓▓▒         ░█      ▒▒      █░   ░█▓██▓█░▒█    ░▓▓█░  ░▓▓█░   ▒▓▓██▓█░   ▒▓     
 ░██▒░  ▒██▒ ▒██░  ░▒█▓                                                                             
         ░▓░ ░▓░   
```

**Review by jmeg-sfy — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]
😸

File: scripts/bootstrap.sh (hunk 1 above)
Comment (mlepage-google) [reply scripts/bootstrap.sh:45]: Ooh nice.

Comment (turon) [reply scripts/bootstrap.sh:45]: Love the logo -- accepting.

**Review by andy31415 — APPROVED** [review]

File: scripts/bootstrap.sh (hunk 1 above)
Comment (jmeg-sfy) [reply scripts/bootstrap.sh]: i did it quickly with an ascii art it is blurry can be better

**Pull request merged**